src/Services/NewsletterService.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Services;
+
+use Blumilk\Meetup\Core\Models\NewsletterSubscriber;
+
+class NewsletterService
+{
+    public function isSubscriber(NewsletterSubscriber $subscriber): string
```

Comment:
I really don't like idea that getter-by-name method is changing something and returning string message. Please refactor this to something more understandable.